packages/vue-components/src/__tests__/Quiz.spec.js

```diff
@@ -51,6 +56,12 @@ const BLANKS_QUESTION = `
 </question>
 `;
 
+// Function to delay test until question is rendered
+function timeoutRenderQuestion() {
+  // eslint-disable-next-line no-promise-executor-return
+  return new Promise(resolve => setTimeout(resolve, 350));
```

Comment:
Is there no way to work around using a timeout?

Something like `waitForSelector(...).toGoMissing()` (imaginary).
Or creating your own simple transition stub.

There are 17 calls here so resolving this should help improve the speed of tests a fair bit. Unless the runner is parallel, I can't remember..

But don't spend too much time on it, 6s to me is still an ok degradation for now. (just create a follow up issue if we can't resolve this).

Reply:
@jovyntls could you help to merge in this MR if it looks good to you? 🙇